.eslintrc.json

```
@@ -1,41 +1,3 @@
 {
-  "root": true,
-  "ignorePatterns": ["projects/**/*"],
-  "overrides": [
-    {
-      "files": ["*.ts"],
-      "parserOptions": {
-        "project": ["tsconfig.json"],
-        "createDefaultProgram": true
-      },
-      "extends": [
-        "plugin:@angular-eslint/recommended",
-        "plugin:@angular-eslint/template/process-inline-templates"
-      ],
-      "rules": {
-        "@angular-eslint/no-empty-lifecycle-method": "off",
-        "@angular-eslint/directive-selector": [
-          "error",
-          {
-            "type": "attribute",
-            "prefix": "app",
-            "style": "camelCase"
-          }
-        ],
-        "@angular-eslint/component-selector": [
-          "error",
-          {
-            "type": "element",
-            "prefix": "app",
-            "style": "kebab-case"
-          }
-        ]
-      }
-    },
-    {
-      "files": ["*.html"],
-      "extends": ["plugin:@angular-eslint/template/recommended"],
-      "rules": {}
-    }
-  ]
+  "extends": "@nhcarrigan/eslint-config-angular"
```

```suggestion
  "extends": "@nhcarrigan/eslint-config-angular",
  "rules": {
    // TODO: Remove once the package is updated.
    "import/no-unresolved": "off"
  }
```

This rule is broken in the package oops - we'll need to fix this over there.

---

Naomi, can you help explain why the module didn't resolve? It'd be helpful for me going forward.

---

If I had to venture a guess, it's either due to the way the deeply nested exports are handled, or the way pnpm resolves the node_modules via symlink.